src/PostgREST/OpenAPI.hs

```diff
@@ -185,7 +185,9 @@ makePathItem (t, cs, _) = ("/" ++ unpack tn, p $ tableInsertable t)
   where
     tOp = (mempty :: Operation)
       & tags .~ Set.fromList [tn]
-      & description .~ tableDescription t
+      -- Use first line of table description as summary; rest as description (if present)
+      & summary .~ (strip . takeWhile (/="\n") <$> tableDescription t)
+      & description .~ mfilter (/="") (strip . dropWhile (/="\n") <$> tableDescription t)
```

Comment:
Also try single quoted \n here.

Also double check that strip eats a leading newline in the description since I think the dropWhile would preserve it.